okhttp/src/main/java/okhttp3/internal/platform/ConscryptPlatform.kt

```diff
@@ -106,18 +106,22 @@ class ConscryptPlatform private constructor() : Platform() {
   }
 
   companion object {
-    fun buildIfSupported(): ConscryptPlatform? = try {
-      // Trigger an early exception over a fatal error, prefer a RuntimeException over Error.
-      Class.forName("org.conscrypt.Conscrypt\$Version")
-
-      when {
-        Conscrypt.isAvailable() && atLeastVersion(2, 1, 0) -> ConscryptPlatform()
-        else -> null
+    val isSupported: Boolean by lazy {
```

Comment:
why lazy?

Reply:
IIRC mainly why run this sort of code unintentionally on other platforms e.g. Android if we aren't using it.  I can simplify.